examples/script/google2016_unbreakable/win.py

```diff
@@ -0,0 +1,62 @@
+from manticore import Manticore
+
+"""
+Leverages Manticore to solve the Google 2016 challenge: 
+unbreakable-enterprise-product-activation
+
+Author: @ctfhacker
+Binary: https://github.com/ctfs/write-ups-2016/blob/master/google-ctf-2016/reverse/unbreakable-enterprise-product-activation-150/unbreakable-enterprise-product-activation.bz2
+"""
+
+m = Manticore('unbreakable')
+
+"""
+These values can be found at 0x4005b8
+"""
+input_addr = 0x6042c0
+num_bytes = 0x43
+
+# Entry point
+@m.hook(0x400729)
+def hook(state):
+    """ CAUTION: HACK """
+    """ From entry point, jump directly to code performing check """
+
+    # Create a symbolic buffer for our input
+    buffer = state.new_symbolic_buffer(0x43)
+
+    # We are given in the challenge that the flag begins with CTF{
+    # So we can apply constraints to ensure that is true
+    state.constraints.add(buffer[0] == ord('C'))
```

Comment:
does this still work if we use `state.add` instead of `state.constraints.add`? if so, let's use that.